testsuite/features/support/commonlib.rb

```diff
@@ -227,6 +227,7 @@ def generate_repository_name(repo_url)
   repo_name.delete_prefix! 'http://download.suse.de/ibs/SUSE:/Maintenance:/'
   repo_name.delete_prefix! 'http://minima-mirror-qam.mgr.prv.suse.net/ibs/SUSE:/Maintenance:/'
   repo_name.sub!('/', '_')
+  repo_name[0...64] # HACK: Due to the 64 characters size limit of a repository label
```

Comment:
@calancha thanks for the tip about the third dot, it worked, you will not go sleep without learn something new.